src/utility.ts

```diff
@@ -93,12 +93,18 @@ export class Utility {
         return crypto.createHash("sha256").update(data).digest("hex");
     }
 
-    public static generateSasTokenForService(iotHubConnectionString: string): string {
+    public static generateSasTokenForService(iotHubConnectionString: string, expiryInHours = 1): string {

```

Comment:
add some tests for utility methosd

Reply:
This API's return would change when it is called, so it would be a little hard to test it in unit test. Or we just verify the format of return value?

Reply:
Added.